cli/cmd/compose/up.go

```diff
@@ -54,25 +53,22 @@ func runUp(ctx context.Context, opts composeOptions, services []string) error {
 		return err
 	}
 
-	_, err = progress.Run(ctx, func(ctx context.Context) (string, error) {
```

Comment:
If we remove this here, we need to add the progres.Run in all backends. Currently this hides all the display in ACI/ECS compose up, as they will fallback to noop progress.

Reply:
so the "WiP" status of this PR :P
I'm looking for a way to concile progress with attached logs

Reply:
seems using progressWriter.Stop() just works
Last issue I need to fix is running `down` on ctrl+c, which never complete